src/cloudai/_core/installables.py

```
@@ -53,24 +53,42 @@ def __str__(self) -> str:
+            # "nvcr.io_nvidia__pytorch__24.02-py3.sqsh"
+
+            DockerImage("registry.example.com:5000#group/project").cache_filename
+            # "registry.example.com_5000_group__project__notag.sqsh"
+
+            DockerImage("/local/cache/image.sqsh").cache_filename
+            # "image.sqsh__notag.sqsh"
+        """
+        reference = self.url.split("://", maxsplit=1)[-1]
+        tag = "notag"
+
+        if reference.startswith("/") or reference.startswith("."):
```

Ruff (0.15.7)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)